### ISSUE TITLE
Data Package viewer

### ISSUE THREAD

**Issue by loleg** [frictionlessdata/forum#70]
The tool for previewing Data Packages at https://data.okfn.org/tools/view is now offline (I'm getting a Cloudflare DNS error). Since we have linked to it quite a lot in the past, I would appreciate a note of explanation, linking at least to the source code in case anyone wants to help the project.

Going forward, I was looking at some alternatives in this space like the [JSON Schema Viewer](https://github.com/jlblcc/json-schema-viewer) (screenshot below), and wondering if there was a roadmap or ideas in this kind of more "utility belt" direction rather than the "preview tool" that this project was until now.

![Screenshot from 2020-11-06 13-34-28](https://user-images.githubusercontent.com/31819/98367094-549f1780-2035-11eb-82fd-d3adf2a644df.png)

**Comment by lwinfree** [comment]
pinging @rufuspollock for help here :-)

**Comment by rufuspollock** [comment]
@loleg i agree that was a very useful tool 😄  - i suspect that OKF folks may have taken down the service and i no longer have access to it (it was running on heroku) ( @lwinfree  could you ask @amercader / @roll about this ... - also it would be nice to implement redirects for data.okfn.org rather than just have cloudflare error and i can make suggestions about what these could be)

@loleg to come to specifically what we do in the near term to replace this

* There is an existing issue about implementing a tool on datahub.io https://github.com/datopian/datahub-qa/issues/274 (the validator is already moved - https://datahub.io/tools/validate)
* It would be easy to put something together relatively rapidly now we have https://github.com/frictionlessdata/frictionless-js going

Aside: re your comment about json schema whilst not directly relevant here as we are rendering the dataset rather than the schema ... it's funny you should ask b/c

* we recently created https://github.com/datopian/jsv
* there was this discussion about json schema tools here https://github.com/frictionlessdata/forum/issues/49

**Comment by roll** [comment]
I don't know what was the initial project but sharing the Data Package Creator link just in case - https://create.frictionlessdata.io/

**Comment by loleg** [comment]
There's an early days screenshot of the viewer here: https://okfnlabs.org/blog/2013/08/06/data-update-2.html
The code lives at https://github.com/okfn/data.okfn.org-new

+1 for frictionless-js !
